podman_cli/podman_images.sh

```diff
@@ -206,7 +206,9 @@ podman_images() {
 
 clean_images_and_containers() {
 
-    read_color "podman rm -a"
+    echo
+    echo_color "Time to clean up!"
+    read_color "podman rm -a -f"
```

Comment:
Is this supposed to be `-a`?

Reply:
nope, but I neglected to add `-f` to the actual command below it so that it matched.